osutil/export_test.go

```diff
@@ -242,24 +242,15 @@ func ParseRawExpandableEnv(entries []string) (ExpandableEnv, error) {
 	return ExpandableEnv{OrderedMap: om}, nil
 }
 
-// this is weird to use in a test, but it is so that we can test the actual
-// implementation of LoadMountInfo, which normally panics during tests if not
-// properly mocked
-func MountInfoMustMock(new bool) (restore func()) {
```

Comment:
thanks for refactoring, this helper was always a bit awkward to use and read, your implementation makes the tests easier to read and write now

Reply:
Thank you. :-)